lib/collective.lib.js

```diff
@@ -137,7 +137,7 @@ export const isEmptyCollectiveLocation = account => {
 
 export const getContributeRoute = collective => {
   let pathname = `/${collective.slug}/donate`;
-  if (collective.settings?.disableCustomContributions) {
+  if (collective.settings?.disableCustomContributions || collective.settings?.disableCryptoContributions) {
```

Comment:
Use `get` and a default, consistency with other code elsewhere.

Reply:
Done. 👍🏽